website/docs/ocl.md

```diff
@@ -22,55 +22,61 @@ The `billing` component defines the integration into cloud provider billing syst
-* `fixedPrice` is the fixed price during the period (the price applied one shot whatever is the service use)
-* `variablePrice` is the price depending of item volume
-* `variableItem` is the item used to calculate the variable price on the period (for instance, the number of instances, the number of transactions, ...)
+* `model` - (Required) Defines the business model (`flat`, `exponential`, ...).
+* `period` - (Required) Defines the rental period (`daily`, `weekly`, `monthly`, `yearly`).
+* `currency` - (Optional) Defines the billing currency (`euro`, `usd`, ...).
+* `fixedPrice` - (Optional) The fixed price during the period (the price applied one shot whatever is the service use).
+* `variablePrice` - (Optional) The price depending of item volume.
+* `variableItem` - (Optional) The item used to calculate the variable price on the period (for instance, the number of instances, the number of transactions, ...).
 
 ### Compute
 
 #### VM
 
 This is the list of VMs used by the service. Each VM has:
 
-* `name` is the name of the VM
-* `type` is the VM type, like `t2.large` for example
-* `platform` is the VM image/platform, like `linux-x64` (to be generic)
-* `vpc` is the name of the VPC defined in the `network` section
-* `subnet` is the name of the Subnet defined in the `network` section
-* `security` is the name of the security defined in the `network` section
-* `storage` is the name of the storage defined in the `storage` section
-* `publicly` is a flag to indicate if the VM should be exposed on Internet (`true`) or only local (`false`)
+* `name` - (Required) The name of the VM.
+* `type` - (Required) The VM type, like `t2.large` for example.
+* `platform` - (Required) The VM image/platform, like `linux-x64` (to be generic).
+* `vpc` - (Required) The `JsonPath` of the VPC.
+* `subnet` - (Required) The `JsonPath` of the Subnet.
+* `security` - (Required) A list of `JsonPath` of `security`.
+* `storage` - (Required) A list of `JsonPath` of the `storage`.
+* `publicly` - (Required) The flag to indicate if the VM should be exposed on Internet (`true`) or only local (`false`).
 
 ### Network
 
 #### VPC
 
 This is the list of VPC defined in the service network. Each VPC has:
 
-* `name` is the name of VPC, used in other OCL elements
-* `cidrs` is a the VPN IP address range
-* `routes` is the routing policy for this VPC
-* `acl` defines the Access Control List for this VPC
+* `name` - (Required) The name of VPC, used in other OCL elements
+* `cidr` - (Required) The VPN IP address range.
 
 #### Subnet
 
 This is the list of subnet defined in the service network. Each subnet has:
 
-* `name` is the name of the subnet, used in other OCL elements
-* `vpc` is the name of the VPC where the subnet is attached
-* `table` is the subnet table
-* `routes` is the routing policy for this subnet
+* `name` - (Required) The name of the subnet, used in other OCL elements.
+* `vpc` - (Required) The `JsonPath` of the VPC where the subnet is attached.
```

Comment:
how about just keeping the orignal description here, as there may be some scenarios that we use an existing vpc instead of creating with the subnet, so here we support string value to existing vpc and `JsonPath` format.

Reply:
I’m afraid that it might be confusing. However, we will pass this string to the CSP’s provider. Then they can do whatever they want.

Reply:
It sounds reasonable for now yes.